packages/teams-js/src/public/media.ts

```diff
@@ -412,30 +414,42 @@
      * @param mediaEvent indicates what the event that needs to be signaled to the host client
      * Optional; @param callback is used to send app if host client has successfully handled the notification event or not
      */
-    protected notifyEventToHost(mediaEvent: MediaControllerEvent, callback?: (err?: SdkError) => void): void {
+    protected notifyEventToHost(mediaEvent: MediaControllerEvent, callback?: (err?: SdkError) => void): Promise<void> {
       ensureInitialized(FrameContexts.content, FrameContexts.task);
-      const err = isApiSupportedOnMobile(nonFullScreenVideoModeAPISupportVersion);
-      if (err) {
-        if (callback) {
-          callback(err);
-        }
-        return;
-      }
+
+      const promise = Promise.resolve().then(() =>
+        throwExceptionIfMobileApiIsNotSupported(nonFullScreenVideoModeAPISupportVersion),
+      );
 
```

Comment:
Do we return Promises if we hit these cases? And do we need to throw on line 430?